frontend/src/lib/components/PropertyKeyInfo.js

```diff
@@ -173,6 +173,52 @@ export const keyMapping = {
             description: 'The initial type of device that was used (first-touch).',
             examples: ['Mobile', 'Tablet', 'Desktop'],
         },
+        $pageview: {
```

Comment:
I am happy to take suggestions for description or example updates to any of these properties